graphql_compiler/query_pagination/parameter_generator.py

```diff
@@ -1,24 +1,53 @@
 # Copyright 2019-present Kensho Technologies, LLC.
+from ..cost_estimation.helpers import is_int_field_type, is_uuid4_type
+from ..cost_estimation.filter_selectivity_utils import convert_uuid_string_to_int, convert_int_to_uuid_string, MIN_UUID_INT, MAX_UUID_INT
 
 
-def generate_parameters_for_parameterized_query(
-    schema_info, parameterized_pagination_queries, num_pages
-):
-    """Generate parameters for the given parameterized pagination queries.
-
-    Args:
-        schema_info: QueryPlanningSchemaInfo
-        parameterized_pagination_queries: ParameterizedPaginationQueries namedtuple, parameterized
-                                          queries for which parameters are being generated.
-        num_pages: int, number of pages to split the query into.
-
-    Returns:
-        two dicts:
-            - dict, parameters with which to execute the page query. The next page query's
-              parameters are generated such that only a page of the original query's result data is
-              produced when executed.
-            - dict, parameters with which to execute the remainder query. The remainder query
-              parameters are generated such that they produce the remainder of the original query's
-              result data when executed.
-    """
+# XXX merge plan:
+# - copy _convert_int_to_field_value and _convert_field_value_to_int from query-pagination branch
+# - implement _get_query_path_endpoint_type properly
+# - land to this PR
+# - find existing filters on the pagination_key
+# - Implement int field paging
+
+
+def _get_query_path_endpoint_type(schema, query_path):
+    current_type = schema.get_type('RootSchemaQuery')
```

Comment:
This might end up being a problem in the long run, since technically I'm not sure anything guarantees this choice of name for the root query type. Consider `schema.get_query_type()` instead, which I believe should be the same thing?